app/app/templates/calculations/punching.html

```diff
@@ -0,0 +1,215 @@
+<!DOCTYPE html>
+<html>
+<head lang="en">
+	<meta charset="UTF-8">
+	<title>Przebicie</title>
+
+	<script src="{{ STATIC_URL }}js/vendor/jquery-1.11.3.min.js"></script>
+	<script src="{{ STATIC_URL }}js/vendor/bootstrap.min.js"></script>
+	<script type="text/javascript" src="http://cdnjs.cloudflare.com/ajax/libs/jquery-cookie/1.4.1/jquery.cookie.min.js"></script>
+
+	<link href="{{ STATIC_URL }}css/vendor/bootstrap.min.css" rel="stylesheet" />
+	<link href="{{ STATIC_URL }}css/style.css" rel="stylesheet" />
+	<link href="//fonts.googleapis.com/css?family=Roboto:300,400,500,700,100" rel="stylesheet" type="text/css">
+
+</head>
+<body>
+
+	<!-- Fixed navbar -->
+    <nav class="navbar navbar-default navbar-fixed-top">
+      <div class="container-fluid">
+        <div class="navbar-header">
+          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+            <span class="sr-only">Toggle navigation</span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+          </button>
+          <a class="navbar-brand" href="http://127.0.0.1:8000/">RCon</a>
+        </div>
+        <div id="navbar" class="collapse navbar-collapse">
+          <ul class="nav navbar-nav">
+            <li class="active"><a href="#">Przebicie</a></li>
+            <li><a href="#">Słup</a></li>
+            <li><a href="#">Wzmacnianie FRP</a></li>
+            <li class="dropdown">
+              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Help <span class="caret"></span></a>
+              <ul class="dropdown-menu">
+                <li><a href="#">About</a></li>
+                <li><a href="#">Version</a></li>
+                <li><a href="#">Contact</a></li>
+                <li role="separator" class="divider"></li>
+                <li class="dropdown-header">Help</li>
+                <li><a href="#">Przebicie - pomoc</a></li>
+                <li><a href="#">Słup - pomoc</a></li>
+              </ul>
+            </li>
+          </ul>
+        </div><!--/.nav-collapse -->
+      </div>
+    </nav>
+
+	<div class="container">
+    	<div class="page-header">
+       	<h1>Przebicie</h1>
+       	<h2>Moduł do obliczania zbrojenia płyty żelbetowej na przebicie</h2>
+    </div>
```

Comment:
Tu są dwa div-y otwarte, a jeden zamknięty. Można to też zrobić, że całość wrzucić do diva o klasie container, czyli wyrzucić div-a spod spodu.